### PR TITLE
[ja] Supervised Learning

### PR DESCRIPTION
**Pull request by ytknzw**
Ready for review

### CODE REVIEW

**Comment by shervinea** [comment]
Awesome to see your pull request @ytknzw, keep up the great work!

**Comment by ytknzw** [comment]
Please review.

**Comment by shervinea** [comment]
Thanks for your work @ytknzw, let's start the review process!

File: ja/cheatsheet-supervised-learning.md [hunk 1]
```diff
@@ -0,0 +1,567 @@
...
+
+**34. Exponential family ― A class of distributions is said to be in the exponential family if it can be written in terms of a natural parameter, also called the canonical parameter or link function, η, a sufficient statistic T(y) and a log-partition function a(η) as follows:**
+
+&#10230;指数分布族 ― 正準パラメータまたはリンク関数とも呼ばれる自然パラメータη、十分統計量T(y)及び対数分配関数a(η)を用いて、次のように表すことのできる一群の分布は指数分布族と呼ばれる：
+
+<br>
+
+**35. Remark: we will often have T(y)=y. Also, exp(−a(η)) can be seen as a normalization parameter that will make sure that the probabilities sum to one.**
+
+&#10230;備考：T(y)=yとすることが多い。また、exp(−a(η))は確率の合計が１になることを担保する正規化定数だと見なせる。
```
Comment (tuananhhedspibk) [inline ja/cheatsheet-supervised-learning.md]: `担保する` -> `確認する`

File: ja/cheatsheet-supervised-learning.md [hunk 2]
```diff
@@ -0,0 +1,567 @@
...
+
+**46. Remark: the line is defined as wTx−b=0.**
+
+&#10230;備考：直線はwTx−b=0と定義する。
+
+<br>
+
+**47. Hinge loss ― The hinge loss is used in the setting of SVMs and is defined as follows:**
+
+&#10230;ヒンジ損失 ― ヒンジ損失はSVMの設定に用いられ、次のように定義される：
```
Comment (tuananhhedspibk) [inline ja/cheatsheet-supervised-learning.md]: `SVM` -> `SVMs`

File: ja/cheatsheet-supervised-learning.md [hunk 3]
```diff
@@ -0,0 +1,567 @@
...
+
+**54. Generative Learning**
+
+&#10230;生成学習
+
+<br>
+
+**55. A generative model first tries to learn how the data is generated by estimating P(x|y), which we can then use to estimate P(y|x) by using Bayes' rule.**
+
+&#10230;生成モデルは、P(x|y)を推定することによりデータがどのように生成されるのかを学習しようとする。それはその後ベイズの定理を用いてP(y|x)を推定することに使える。
```
Comment (tuananhhedspibk) [inline ja/cheatsheet-supervised-learning.md]: `推定することに使える` -> `推定するために使える`

File: ja/cheatsheet-supervised-learning.md [hunk 4]
```diff
@@ -0,0 +1,567 @@
...
+
+**56. Gaussian Discriminant Analysis**
+
+&#10230;ガウシアン判別分析
+
+<br>
+
+**57. Setting ― The Gaussian Discriminant Analysis assumes that y and x|y=0 and x|y=1 are such that:**
+
+&#10230;前提 ― ガウシアン判別分析はyとx|y=0とx|y=1は次のようであることを前提とする：
```
Comment (tuananhhedspibk) [inline ja/cheatsheet-supervised-learning.md]: `前提` -> `設定`

File: ja/cheatsheet-supervised-learning.md [hunk 5]
```diff
@@ -0,0 +1,567 @@
...
+
+**59. Naive Bayes**
+
+&#10230;ナイーブベイズ
+
+<br>
+
+**60. Assumption ― The Naive Bayes model supposes that the features of each data point are all independent:**
+
+&#10230;仮定 ― ナイーブベイズモデルは、個々のデータ点の特徴量が全て独立であると仮定する：
```
Comment (tuananhhedspibk) [inline ja/cheatsheet-supervised-learning.md]: `特徴量` -> `特徴`

File: ja/cheatsheet-supervised-learning.md [hunk 6]
```diff
@@ -0,0 +1,567 @@
...
+
+<br>
+
+**61. Solutions ― Maximizing the log-likelihood gives the following solutions, with k∈{0,1},l∈[[1,L]]**
+
+&#10230;解 ― 対数尤度を最大にすると次の解を得る。ただし、k∈{0,1},l∈[[1,L]]とする。
```
Comment (tuananhhedspibk) [inline ja/cheatsheet-supervised-learning.md]: `対数尤度を最大にすると次の解を得る。ただし、k∈{0,1},l∈[[1,L]]とする` -> `k∈{0,1},l∈[[1,L]]で対数尤度を最大にすると次の解を得る`

File: ja/cheatsheet-supervised-learning.md [hunk 7]
```diff
@@ -0,0 +1,567 @@
...
+
+**62. Remark: Naive Bayes is widely used for text classification and spam detection.**
+
+&#10230;備考：ナイーブベイズはテキスト分類やスパム検知に幅広く使われている。
+
+<br>
+
+**63. Tree-based and ensemble methods**
+
+&#10230;ツリーとアンサンブル学習
```
Comment (tuananhhedspibk) [inline ja/cheatsheet-supervised-learning.md]: `ツリー` -> `ツリーベース`

File: ja/cheatsheet-supervised-learning.md [hunk 8]
```diff
@@ -0,0 +1,567 @@
...
+
+**64. These methods can be used for both regression and classification problems.**
+
+&#10230;これらの方法は回帰と分類問題の両方に使える。
+
+<br>
+
+**65. CART ― Classification and Regression Trees (CART), commonly known as decision trees, can be represented as binary trees. They have the advantage to be very interpretable.**
+
+&#10230;CART ― 分類・回帰ツリー (CART)は、一般には決定木として知られ、二分木として表される。非常に解釈しやすいという利点がある。
```
Comment (tuananhhedspibk) [inline ja/cheatsheet-supervised-learning.md]: `決定木` -> `決定ツリー`

File: ja/cheatsheet-supervised-learning.md [hunk 9]
```diff
@@ -0,0 +1,567 @@
...
+
+**69. [Adaptive boosting, Gradient boosting]**
+
+&#10230;[適応的ブースティング, 勾配ブースティング]
+
+<br>
+
+**70. High weights are put on errors to improve at the next boosting step**
+
+&#10230;次のブースティングステップにて改善すべき誤分類に大きい重みが課される。
```
Comment (tuananhhedspibk) [inline ja/cheatsheet-supervised-learning.md]: `改善すべき` -> `改善するための`

File: ja/cheatsheet-supervised-learning.md [hunk 10]
```diff
@@ -0,0 +1,567 @@
...
+
+**3. Given a set of data points {x(1),...,x(m)} associated to a set of outcomes {y(1),...,y(m)}, we want to build a classifier that learns how to predict y from x.**
+
+&#10230;入力が{x(1),...,x(m)}, 出力が{y(1),...,y(m)}であるとき, xからyを予測する分類器を構築したい。
+
+<br>
+
+**4. Type of prediction ― The different types of predictive models are summed up in the table below:**
+
+&#10230;予測の種類 ― 様々な種類の予測モデルは下表に集約される：
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Translating 'different' as '他' or '他の' could mean differently from the original text means. Also, "他種類" isn't a natural phrase in Japanese.

File: ja/cheatsheet-supervised-learning.md [hunk 11]
```diff
@@ -0,0 +1,567 @@
...
+
+**6. [Continuous, Class, Linear regression, Logistic regression, SVM, Naive Bayes]**
+
+&#10230;連続値, クラス, 線形回帰, ロジスティック回帰, SVM, ナイーブベイズ
+
+<br>
+
+**7. Type of model ― The different models are summed up in the table below:**
+
+&#10230;モデルの種類 ― 様々な種類のモデルは下表に集約される：
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Same as this: https://github.com/shervinea/cheatsheet-translation/pull/144/files/3cfde5a1d6bced3dc993c6655580efd478132c8a#r314659845

File: ja/cheatsheet-supervised-learning.md [hunk 12]
```diff
@@ -0,0 +1,567 @@
...
+
+**8. [Discriminative model, Generative model, Goal, What's learned, Illustration, Examples]**
+
+&#10230;判別モデル, 生成モデル, 目的, 学習対象, イメージ図, 例
+
+<br>
+
+**9. [Directly estimate P(y|x), Estimate P(x|y) to then deduce P(y|x), Decision boundary,  	Probability distributions of the data, Regressions, SVMs, GDA, Naive Bayes]**
+
+&#10230;P(y|x)を直接推定する, P(y|x)を推測するためにP(x|y)を推定する, 決定境界, データの確率分布, 回帰, SVM, GDA, ナイーブベイズ
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: In English, 'SVMs' is plural of 'SVM'. On the other hand, in Japanese, plural of 'SVM' is 'SVM'.

File: ja/cheatsheet-supervised-learning.md [hunk 13]
```diff
@@ -0,0 +1,567 @@
...
+
+**17. Remark: Stochastic gradient descent (SGD) is updating the parameter based on each training example, and batch gradient descent is on a batch of training examples.**
+
+&#10230;備考：確率的勾配降下法(SGD)は学習標本全体を用いてパラメータを更新し、バッチ勾配降下法は学習標本の各バッチ毎に更新する。
+
+<br>
+
+**18. Likelihood ― The likelihood of a model L(θ) given parameters θ is used to find the optimal parameters θ through maximizing the likelihood. In practice, we use the log-likelihood ℓ(θ)=log(L(θ)) which is easier to optimize. We have:**
+
+&#10230;尤度 ― パラメータをθとすると、あるモデルの尤度L(θ)を最大にすることにより最適なパラメータを求められる。実際には、最適化しやすい対数尤度ℓ(θ)=log(L(θ))を用いる。すなわち：
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: 'is easier to optimize' -> '最適化することがより簡単である' -> '最適化しやすい'
Also, '最適化しやすいための' isn't natural.

File: ja/cheatsheet-supervised-learning.md [hunk 14]
```diff
@@ -0,0 +1,567 @@
...
+
+<br>
+
+**19. Newton's algorithm ― The Newton's algorithm is a numerical method that finds θ such that ℓ′(θ)=0. Its update rule is as follows:**
+
+&#10230;ニュートン法 ― ニュートン法とはℓ′(θ)=0となるθを求める数値計算アルゴリズムである。そのパラメータ更新は次のように行われる：
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Review accepted.

[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Review accepted.

File: ja/cheatsheet-supervised-learning.md [hunk 15]
```diff
@@ -0,0 +1,567 @@
...
+
+<br>
+
+**20. Remark: the multidimensional generalization, also known as the Newton-Raphson method, has the following update rule:**
+
+&#10230;備考：高次元正則化またはニュートン-ラフソン法ではパラメータ更新は次のように行われる：
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Review accepted.

File: ja/cheatsheet-supervised-learning.md [hunk 16]
```diff
@@ -0,0 +1,567 @@
...
+
+**24. Normal equations ― By noting X the matrix design, the value of θ that minimizes the cost function is a closed-form solution such that:**
+
+&#10230;正規方程式 ― Xを行列とすると、コスト関数を最小化するθの値は次のような閉形式の解である：
+
+<br>
+
+**25. LMS algorithm ― By noting α the learning rate, the update rule of the Least Mean Squares (LMS) algorithm for a training set of m data points, which is also known as the Widrow-Hoff learning rule, is as follows:**
+
+&#10230;最小2乗法 ― 学習率をαとすると、m個のデータ点からなる学習データに対する最小2乗法（LMSアルゴリズム）によるパラメータ更新は次のように行われ、これはウィドロウ-ホフの学習規則としても知られている：
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Review accepted.

File: ja/cheatsheet-supervised-learning.md [hunk 17]
```diff
@@ -0,0 +1,567 @@
...
+
+<br>
+
+**26. Remark: the update rule is a particular case of the gradient ascent.**
+
+&#10230;備考：この更新は勾配上昇法の特殊な例である。
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Review accepted.

File: ja/cheatsheet-supervised-learning.md [hunk 18]
```diff
@@ -0,0 +1,567 @@
...
+
+**31. Remark: there is no closed form solution for the case of logistic regressions.**
+
+&#10230;備考：ロジスティック回帰については閉形式の解は存在しない。
+
+<br>
+
+**32. Softmax regression ― A softmax regression, also called a multiclass logistic regression, is used to generalize logistic regression when there are more than 2 outcome classes. By convention, we set θK=0, which makes the Bernoulli parameter ϕi of each class i equal to:**
+
+&#10230;ソフトマックス回帰 ― ソフトマックス回帰は、多クラス分類ロジスティック回帰とも呼ばれ、3個以上の結果クラスがある場合にロジスティック回帰を一般化するためのものである。慣習的に、θK=0とすると、各クラスiのベルヌーイ分布のパラメータϕiは次と等しくなる：
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Here we talk about number of classes. 'more than 2 classes' means '3 or more classes' and 2 is excluded in this case. It means the same as '3個以上のクラス' in Japanese.

File: ja/cheatsheet-supervised-learning.md (hunk 1 above)
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: 'to make sure that ...' should mean 'to check that ...' or '...であることを確認する'. On the contrary, here it means 'to set or fix that ...' that should be translated as '...であることを担保する' than '確認する'.

File: ja/cheatsheet-supervised-learning.md [hunk 19]
```diff
@@ -0,0 +1,567 @@
...
+
+**9. [Directly estimate P(y|x), Estimate P(x|y) to then deduce P(y|x), Decision boundary,  	Probability distributions of the data, Regressions, SVMs, GDA, Naive Bayes]**
+
+&#10230;P(y|x)を直接推定する, P(y|x)を推測するためにP(x|y)を推定する, 決定境界, データの確率分布, 回帰, SVM, GDA, ナイーブベイズ
+
+<br>
+
+**10. Notations and general concepts**
+
+&#10230;記法と概念
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Review accepted.

File: ja/cheatsheet-supervised-learning.md (hunk 2 above)
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: https://github.com/shervinea/cheatsheet-translation/pull/144/files/3cfde5a1d6bced3dc993c6655580efd478132c8a#r314661353

File: ja/cheatsheet-supervised-learning.md [hunk 20]
```diff
@@ -0,0 +1,567 @@
...
+
+**15. Cost function ― The cost function J is commonly used to assess the performance of a model, and is defined with the loss function L as follows:**
+
+&#10230;コスト関数 ― コスト関数Jは一般的にモデルの性能を評価するために用いられ、損失関数をLとして次のように定義される：
+
+<br>
+
+**16. Gradient descent ― By noting α∈R the learning rate, the update rule for gradient descent is expressed with the learning rate and the cost function J as follows:**
+
+&#10230;勾配降下法 ― 学習率をα∈Rとし、勾配降下法におけるパラメータの更新は学習率とコスト関数Jを用いて次のように行われる：
```
[the remark this answers is not in the document]
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Review accepted.

File: ja/cheatsheet-supervised-learning.md (hunk 3 above)
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: Review accepted.

File: ja/cheatsheet-supervised-learning.md (hunk 5 above)
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: In machine learning context, 'feature' is usually translated as '特徴量'.

File: ja/cheatsheet-supervised-learning.md (hunk 4 above)
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: '前提条件' could be better than '前提' and '設定'.

File: ja/cheatsheet-supervised-learning.md (hunk 6 above)
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: 'k∈{0,1},l∈[[1,L]]' are conditions for the solutions, not for log-likelihood maximisation.

File: ja/cheatsheet-supervised-learning.md (hunk 8 above)
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: 'decision tree' is usually translated as '決定木'. https://ja.wikipedia.org/wiki/%E6%B1%BA%E5%AE%9A%E6%9C%A8

File: ja/cheatsheet-supervised-learning.md (hunk 7 above)
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: I change 'ツリー' to '決定木' based on the context.

File: ja/cheatsheet-supervised-learning.md (hunk 9 above)
Comment (ytknzw) [reply ja/cheatsheet-supervised-learning.md]: 'errors to improve at the next boosting step' = 'errors that need to be improved at the next boosting step'

**Comment by ytknzw** [comment]
@shervinea I improved translations. Thank you @tuananhhedspibk !

**Comment by shervinea** [comment]
Thank you @ytknzw and @tuananhhedspibk!

**Pull request merged**